### PR TITLE
Register and Login option removed of the logged in person from Sitemap

### LINKED ISSUE #1988: Register and Login option should be removed of the logged in person from Sitemap
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behaviour:**

Register and Login option should be removed of the logged in person from Sitemap

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by rajvaibhavdubey**
Fixes #1988 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Register and Login option removed of the logged in person from Sitemap

### CODE REVIEW

File: frontend/app/templates/sitemap.hbs [hunk 1]
```diff
@@ -19,26 +19,23 @@
...
-                {{init-cap 'forgot password'}}
-            {{/link-to}}
-            <br>
             {{#if session.isAuthenticated}}
-              {{#link-to 'settings' class='links'}}
+                {{#link-to 'settings' class='links'}}
                     {{init-cap 'view profile'}}
                 {{/link-to}}
+                <br>
+                {{#link-to 'forgotpwd' class='links'}}
```
Comment (sk9331657) [inline frontend/app/templates/sitemap.hbs]: @rajvaibhavdubey  Forgot password should be available even if users is not logged in .

File: frontend/app/templates/sitemap.hbs [hunk 2]
```diff
@@ -19,26 +19,23 @@
...
-            {{#link-to 'signup' class='links'}}
-                {{init-cap 'register'}}
-            {{/link-to}}
-            <br>
-            {{#link-to 'login' class='links'}}
-                {{init-cap 'login'}}
-            {{/link-to}}
-            <br>
-            {{#link-to 'forgotpwd' class='links'}}
-                {{init-cap 'forgot password'}}
```
Comment (abishekvashok) [inline frontend/app/templates/sitemap.hbs]: Please don't remove this

**Comment by mariobehling** [comment]
Closing this due to inactivity.

**Comment by rajvaibhavdubey** [comment]
@mariobehling This PR has been resolved and can be successfully merged.

**Comment by sk9331657** [comment]
@rajvaibhavdubey I think this is  not required as user will be automatically redirected.

**Pull request closed without merging**